src/rard/research/models/fragment.py

```diff
@@ -75,6 +74,7 @@ def definite_work_and_book_links(self):
         return (
```

Comment:
Does it matter that we overhaul this in the next PR? (The definite/possible grouping)

Reply:
I think it's best to keep interdependencies between PRs to a minimum, so we know this version "works".